### PR TITLE
Fix typo in troubleshooting-nodejs, adopters and malabi

### PR DESCRIPTION
**Pull request by ydFu**
The reasons for this correction are as follows:

- Rectification of some grammatical errors, such as the addition of periods or the removal of redundant words.
- Change files:
  - content/en/blog/2021/malabi.md
  - content/en/ecosystem/adopters.md
  - content/en/ecosystem/adopters.md

### CODE REVIEW

File: content/en/ecosystem/adopters.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ To have your organization listed, submit a PR with an entry added to the
 The entry must include the following[^grace-period-2024-01-01]:
 
 - Link to a blog post that describes how your organization makes use of
-  OpenTelemetry
+  OpenTelemetry.
 - GitHub handle or email address as a point of contact so that we can reach out
-  in case we have questions
+  in case we have questions.
```
Comment (svrnm) [inline content/en/ecosystem/adopters.md:26]: @chalin @cartermp my feeling for the English language is not good enough here: are items of an itemized list of non-sentences terminated by a dot? If so, we have to change this on a few other pages as well.

**Comment by svrnm** [comment]
@ydFu thank you for making the effort to reporting these language issues!

File: content/en/ecosystem/adopters.md (hunk 1 above)
Comment (theletterf) [reply content/en/ecosystem/adopters.md:26]: Usually, one doesn't end with punctuation any list items that are incomplete sentences. Then again, you might want to do it for consistency if items are mixed. The Splunk Style Guide says:

> If the list items are complete sentences, include end punctuation.
> If the list items are characters, words, or phrases, don't include end punctuation.
> Use parallel sentence construction in list items. For example, start all list items with verbs, or end all list items with punctuation.

Comment (svrnm) [reply content/en/ecosystem/adopters.md:26]: Ok, this sounds like a reasonable norm to me (since I would do it that way as well in German) 👍

Comment (ydFu) [reply content/en/ecosystem/adopters.md]: Thanks for the reminder, I also agree with the suggested guidelines above.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
thank you @ydFu !

**Pull request merged**